### PR TITLE
Update latest embedded c release notes for November

### PR DESCRIPTION
**Pull request by ahsonkhan**

### CODE REVIEW

File: releases/2022-11/c.md [hunk 1]
```diff
@@ -34,9 +27,16 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 ## Release highlights
 
-### _Package name_
+### azure-sdk-for-c [Changelog](https://github.com/Azure/azure-sdk-for-c/blob/1.4.0/CHANGELOG.md#140-2022-11-11)
```
Comment (ahsonkhan) [inline releases/2022-11/c.md:30]: This link will be valid after the tag has been released.

File: releases/2022-11/c.md [hunk 2]
```diff
@@ -34,9 +27,16 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+
+#### Features Added
+
+- [[#2329](https://github.com/Azure/azure-sdk-for-c/pull/2329)] Add Base64 URL decoder.
+- Added support in `az_json.h` to unescape JSON string tokens within an `az_span` using the `az_json_string_unescape()` API.
```
Comment (ahsonkhan) [inline releases/2022-11/c.md:35]: Waiting for https://github.com/Azure/azure-sdk-for-c/pull/2375 to be merged, along with a Changelog entry for the new IoT ADU feature, which will be added and highlighted here.

File: releases/2022-11/c.md [hunk 3]
```diff
@@ -10,22 +10,15 @@ The Azure SDK team is pleased to make available the November 2022 client library
...
+- Azure Core and Azure IoT
 
 ## Installation Instructions
 
 To install any of our packages, copy and paste the following commands into a terminal:
 
 ```bash
-$> 
+git clone https://github.com/Azure/azure-sdk-for-c
+git checkout <tag_name>
```
Comment (danewalton) [inline releases/2022-11/c.md:21]:
```suggestion
git clone https://github.com/Azure/azure-sdk-for-c
cd azure-sdk-for-c
git checkout <tag_name>
```

**Review by danewalton — APPROVED** [review]

File: releases/2022-11/c.md (hunk 1 above)
Comment (ahsonkhan) [reply releases/2022-11/c.md:31]: Done:
https://github.com/Azure/azure-sdk-for-c/blob/1.4.0/CHANGELOG.md#140-2022-11-11

**Pull request merged**